### ISSUE TITLE
Reports Total table squished

### ISSUE THREAD

**Issue by connorshea** [salesagility/SuiteCRM#7979]
#### Issue

The Report Total field became much smaller at some point and looks weird compared to the main Reports table (I don't think this is the same as #7964?).

See also https://suitecrm.com/suitecrm/forum/feedback/24368-problem-in-the-report-module-with-columns-width-in-the-total-row-of-a-report-table

#### Expected Behavior
The report's Total table would span the full width of its container.

#### Actual Behavior
It's squished into a very small width, only as long as the column title needs.

![Screen Shot 2019-10-04 at 3 13 01 PM](https://user-images.githubusercontent.com/2977353/66240294-7e12a480-e6b9-11e9-9e44-80328c069119.png)

This was posted in the forum thread as an example:

![image](https://user-images.githubusercontent.com/2977353/66240351-a9958f00-e6b9-11e9-8c5d-a113a75741ce.png)

#### Possible Fix
It seems like we can mostly fix it by adding these classes and width 100% to the table (right now it's literally just a `<table>` tag with no attributes):

```html
<table width="100%" class="list view table-responsive aor_reports">
  <!-- table stuff -->
</table>
```

This is what it looks like with that change:

![Screen Shot 2019-10-04 at 3 18 26 PM](https://user-images.githubusercontent.com/2977353/66240573-448e6900-e6ba-11e9-88fd-6d425d0ce40d.png)

It has extra columns (same number as the report?) for some reason, which is why the header text is weirdly aligned.

#### Steps to Reproduce
1. Create a Report in the AOR_Reports module, with a 'total' field.
2. View the report.
3. See that the total field's table is much smaller than the main table.

#### Context
It looks ugly :)

#### Your Environment
* SuiteCRM Version used: 7.10.19
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome 79, Firefox 69
* Environment name and version (e.g. MySQL, PHP 7): MySQL 5.6, PHP 7.2
* Operating System and version (e.g Ubuntu 16.04): macOS Mojave 10.14.6

**Comment by connorshea** [comment]
Fixing this will probably require `getTotalTable()` in `modules/AOR_Reports/AOR_Report.php` to be refactored somewhat heavily (or we could just hack around it, I guess).

https://github.com/salesagility/SuiteCRM/blob/54c574bdc681cf74d61d64effb8c9bc77461a122/modules/AOR_Reports/AOR_Report.php#L950-L962

See also #7965 for a fix for a different problem, but in the same file.

**Comment by kniziol** [comment]
Hi @connorshea. If no-one is working on this issue, feel free to assign me. I can fix it, if the application will start on my local machine without any efforts.

**Comment by Dillon-Brown** [comment]
Hi @kniziol, apologies for missing the label. This issue has a fix proposed here: #7984
